### ISSUE TITLE
kitti datasets

### ISSUE THREAD

**Issue by Baboom-l** [mileyan/Pseudo_Lidar_V2#16]
Hi! 
I want to know  python ./src/preprocess/generate_depth_map.py --data_path path-to-KITTI/ --split_file ./split/trainval.txt

 --split_file ./split/trainval.txt  This file path does that mean I need to set up my own validation set from training set？
